olive/passes/onnx/conversion.py

```diff
@@ -137,19 +137,16 @@ def _convert_model_on_device(
             if isinstance(dummy_inputs, dict):
                 dummy_input_keys = set(dummy_inputs.keys())
 
-                # handle dummy inputs for hf model with past, which has past_key_values
+                # handle dummy inputs for model with past, which has past_key_values
                 # match input names in `past_key_values.(hidden_layer_num).(key|value)` pattern
-                from transformers.modeling_utils import PreTrainedModel
-
-                if issubclass(type(pytorch_model), PreTrainedModel):
```

Comment:
We use the same cache format as huggingface here, so we can leverage this logic without restricting it only for `PreTrainedModel`